### ISSUE TITLE
resource genesyscloud_tf_export >> replace_with_datasource = ["genesyscloud_group::group"] fails

### ISSUE THREAD

**Issue by juliomasuda** [MyPureCloud/terraform-provider-genesyscloud#1133]
I am using the example described at https://registry.terraform.io/providers/MyPureCloud/genesyscloud/latest/docs/resources/tf_export#example-usage
and the output is:
genesyscloud_tf_export.export: Creating...
│ Error: failed to get page of journey action maps error: API Error: 403 - Unable to perform the requested action. You must have at least one of the following permissions assigned: [journey:actionmap:view] (c5fad4cb-d540-4913-be4a-761b3dec1281)
│ To continue exporting other resources in spite of this error, set the 'log_permission_errors' attribute to 'true'
│   with genesyscloud_tf_export.export,
│   on main.tf line 41, in resource "genesyscloud_tf_export" "export":
│   41: resource "genesyscloud_tf_export" "export" {
│{"resourceName":"genesyscloud_journey_action_map","method":"GET","path":"/api/v2/journey/actionmaps","statusCode":403,"errorMessage":"API
│ Error: 403 - Unable to perform the requested action. You must have at least one of the following permissions
│ assigned: [journey:actionmap:view]
│ (c5fad4cb-d540-4913-be4a-761b3dec1281)","correlationId":"c5fad4cb-d540-4913-be4a-761b3dec1281"}
In fact, the "[journey:actionmap:view]" permission does not exist in my organization, but I have enabled all permissions on "journey" for this OAuth user.
I am using  terraform - v.1.8.5
terraform {
  required_providers {
    genesyscloud = {
      source = "MyPureCloud/genesyscloud"
      version = "1.40.1"
    }}}
If I comment the line: 
## enable_dependency_resolution = true
the error message changes to "workitems":
genesyscloud_tf_export.export: Creating...
│ Error: Failed to get task management worktype error: failed to get worktypes: API Error: 403 - Unable to perform the requested action. You must have permission workitems:worktype:view in at least one division. (c5a19e70-2faf-4e32-a61c-12368e32676b)
│ To continue exporting other resources in spite of this error, set the 'log_permission_errors' attribute to 'true'
│   with genesyscloud_tf_export.export,
│   on main.tf line 41, in resource "genesyscloud_tf_export" "export":
│   41: resource "genesyscloud_tf_export" "export" {
│{"resourceName":"genesyscloud_task_management_worktype","method":"POST","path":"/api/v2/taskmanagement/worktypes/query","statusCode":403,"errorMessage":"API
│ Error: 403 - Unable to perform the requested action. You must have permission workitems:worktype:view in at least one
│ division. (c5a19e70-2faf-4e32-a61c-12368e32676b)","correlationId":"c5a19e70-2faf-4e32-a61c-12368e32676b"}
In fact, my organization does not have any workitem permissions available in the permissions tab.
It appears this resource is looking for resources not enabled in my organization.

**Comment by charliecon** [comment]
Hi @juliomasuda 

Have you tried using the `log_permission_errors` field? This should tell the exporter to resume in spite of permissions errors. 

-Charlie

**Comment by juliomasuda** [comment]
Hi @charliecon,
After hours of studying this subject and several attempts I managed to achieve my goal.
Anyway, this usage example didn't work for me, maybe it should be updated for other people to use.
After hours studying this subject and several attempts I managed to achieve my goal.
Anyway, this usage example didn't work for me, maybe it should be updated for other people to use.
Regardless, I appreciate your comment.
